podcasts/DiscoverPodcastSearchResultsController.swift

```diff
@@ -133,12 +133,16 @@ class DiscoverPodcastSearchResultsController: UIViewController, UITableViewDeleg
 
     // MARK: - Search
 
-    func clearSearchResults() {
+    func clearSearch() {
         searchResults.removeAll()
         searchState = .notStarted
         searchResultsTable.reloadData()
     }
 
+    func performLocalSearch(searchTerm: String) {}
+
+    func performRemoteSearch(searchTerm: String, completion: @escaping (() -> Void)) { }
+
```

Comment:
I think all in all the problem remains the same. 😬

My plan, for the future, is to have a single protocol with one clear and one search function. Is is this way now because of the current implementation, now the new one.

Anyway, I changed in b181532378451279d93e3a7a468d4b762b6a681f